Dockerfile

```diff
@@ -293,6 +297,10 @@ RUN update-alternatives --set iptables  /usr/sbin/iptables-legacy  || true \
 
 RUN pip3 install yamllint==1.16.0
 
+# hack/dind switches to hack/dind-systemd when $DOCKER_SYSTEMD=1 is set
+ADD https://raw.githubusercontent.com/AkihiroSuda/containerized-systemd/b70bac0daeea120456764248164c21684ade7d0d/docker-entrypoint.sh /go/src/github.com/docker/docker/hack/dind-systemd
```

Comment:
Should we add this file to this repository, or are there specific reasons to keep it separate?

Reply:
Wanted to avoid diverging

Reply:
I'd prefer to do this with `RUN curl` instead, otherwise this will always hit github on build, I think.

Reply:
updated to use curl